subprojects/core-api/src/main/java/org/gradle/api/artifacts/dsl/RepositoryHandler.java

```diff
@@ -43,7 +43,7 @@ public interface RepositoryHandler extends ArtifactRepositoryContainer {
      *
      * The following parameter are accepted as keys for the map:
      *
-     * <table summary="Shows property keys and associated values">
```

Comment:
❓ Why removing summary here?

Reply:
The summary breaks the Javadoc rendering in HTML5